packages/@aws-cdk/aws-codepipeline-actions/lib/codebuild/build-action.ts

```diff
@@ -77,6 +77,13 @@ export interface CodeBuildActionProps extends codepipeline.CommonAwsActionProps
    * @default - No additional environment variables are specified.
    */
   readonly environmentVariables?: { [name: string]: codebuild.BuildEnvironmentVariable };
+
+  /**
+   * Trigger a batch build.
+   *
+   * @default false
+   */
+  readonly batchEnabled?: boolean;
```

Comment:
I'm not a huge fan of this name :/ .

`executeBatchBuild` perhaps? I'm also open to suggestions for a great name 🙂.

Reply:
Sure. I just went with the same name as the config. Will update